src/pages/questions.js

```diff
@@ -20,18 +23,25 @@ export default function QuestionPage() {
 					content="Here you can see all the Questions"
 				/>
 			</Helmet>
+
 			<Button
 				greyButton={true}
 				onClick={() => {
 					setFilter(!filter);
 				}}
 			>
-				{!filter ? `Show Closed` : `Show Open`}
+				{filter ? `Show All Questions` : `Show My Questions`}
 			</Button>
 			<article>
 				{questions.map(question => {
-					if (question.answered === filter) {
-						return <QuestionComponent key={question.id} question={question} />;
+					if (filter ? question.userID === activeUser.userID : true) {
```

Comment:
This is rather complicated to work with, a ternary in an `if` statement. Could you simplify this a bit? What do you want to archive with this?
If the filter is false, all questions will be shown, if it is true then only your own? 🤔 

I personally prefer to have such logic as simple as possible. To me it is not quite clear what is happening here and that might cause issues in the future - even to yourself :)

EDIT: I saw not, you are only changing the styles, right?

Reply:
I have two ternary operators inside my map function.
The first in line 37 where i check if i have set a filter, and then im checking if this is "my" question.
If no filter is set, the ternary returns true, so that every questions gets rendered.

I thought about some different solutions, but so far this is the "best" i´ve come up with.

The second ternary is inside the component itself. This is to change the questions card color, so that i can recognize my own questions.